src/parser/parser_visitor.py

```diff
@@ -381,6 +390,24 @@ def visitFromClause(self, ctx: evaql_parser.FromClauseContext):
 
         return {"from": from_table, "where": where_clause}
 
+    def visitOrderByClause(self, ctx: evaql_parser.OrderByClauseContext):
+        orderby_clause_data = []
+        # [(TupleValueExpression #1, ASC), (TVE #2, DESC), ...]
+        for expression in ctx.orderByExpression():
+            orderby_clause_data.append(self.visitOrderByExpression(expression))
```

Comment:
Don't explicitly call a visit function. We don't want to disturb the auto-generated call stack. Instead, replace it will `self.visit(expression)`